packages/story-editor/src/components/inspector/inspectorProvider.js

```diff
@@ -154,6 +159,13 @@ function InspectorProvider({ inspectorTabs, children }) {
     });
   }
 
+  if (inspectorTabs?.document && isUpdatedPublishModalEnabled) {
```

Comment:
Once this isn't under a feature flag it can get added to the same check on line 155, for now I did it this way because I didn't want to nest a check for the feature, this felt cleaner.